packages/ui-components/src/components/tree/tree.types.ts

```diff
@@ -0,0 +1,15 @@
+import { EIconName, EOtherIconName } from '../icon/icon.types';
+import { ETreeItemStates } from '../tree-item/tree-item.types';
+
+export interface ITreeNodeItem {
+	id: string; // Should be unique
+	label?: string;
+	additionalLabel?: string;
+	placeholder?: string;
+	icon?: EIconName | EOtherIconName;
```

Comment:
Should we create a type to icons that combine both enums? Like:

```
export const EIcon = { ...EIconName, ...EOtherIconName };
export type EIcon = typeof EIcon;
```

Reply:
It is better to do it in another pull request because it is used in another places